package.json

```diff
@@ -124,6 +124,7 @@
     "test": "make test",
     "test-client": "NODE_ENV=test NODE_PATH=test:client test/runner.js",
     "test-server": "NODE_ENV=test NODE_PATH=test:server:client test/runner.js",
+    "test-tests": "NODE_ENV=test NODE_PATH=test:test/test test/runner.js",
```

Comment:
Thanks for writing it down, especially after it was merged - this isn't sarcastic at all, by the way - I appreciate the care.